requirements.conda.yml

```diff
@@ -0,0 +1,22 @@
+name: tiatoolbox
+channels:
+  - conda
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.6
+  - setuptools==45.1.0
+  - Click==7.0
+  - cython=0.29.15
+  - h5py=2.8.0
+  - matplotlib-base=3.1.3
+  - numpy=1.18.1
+  - opencv=4.2
+  - pillow=7.0.0
+  - pip=20.0.2
+  - pyyaml=5.3.1
+  - requests=2.23.0
+  - pathos==0.2.5
+  - pip:
+    - openslide-python==1.1.1
+
```

Comment:
can't see this file in the branch?

Reply:
Here https://github.com/TIA-Lab/tiatoolbox/blob/feature-slide-info/requirements.conda.yml